ospfd/ospf_zebra.c

```diff
@@ -1354,7 +1602,7 @@ void ospf_zebra_init(struct thread_master *master, unsigned short instance)
 {
 	/* Allocate zebra structure. */
 	zclient = zclient_new(master, &zclient_options_default);
-	zclient_init(zclient, ZEBRA_ROUTE_OSPF, instance, &ospfd_privs);
+	zclient_init(zclient, ZEBRA_ROUTE_OSPF, 0, &ospfd_privs);
```

Comment:
Please drop this change because it would break MI support.

Reply:
done